.builds/freebsd.yml

```diff
@@ -24,7 +24,7 @@ tasks:
   - clippy: |
       cd alacritty
       rustup component add clippy
-      cargo clippy --all-targets
+      cargo clippy
```

Comment:
I'm not aware of any benefit of passing `--all-targets` in our current version of Alacritty. Since everything is in the workspace clippy just warns about everything by default as far as I can tell.

Reply:
Hold on this please, I think I know why I am using this.

Reply:
Correction, this is necessary for running clippy against tests.